bucket/npiperelay.json

```
@@ -0,0 +1,35 @@
+{
+    "version": "v0.1.0",
+    "description": "npiperelay allows you to access Windows named pipes from WSL",
+    "homepage": "https://github.com/jstarks/npiperelay",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/jstarks/npiperelay/releases/download/v0.1.0/npiperelay_windows_amd64.zip",
+            "hash": "6b9ef61ffd17c03507a9a3d54d815dceb3dae669ac67fc3bf4225d1e764ce5f6"
+        },
+        "32bit": {
+            "url": "https://github.com/jstarks/npiperelay/releases/download/v0.1.0/npiperelay_windows_386.zip",
+            "hash": "600e3f94f0d9f48e68b5a9276849d00d658ae9c18c81541463ed11b1dfe6c998"
+        }
+    },
+    "extract_to": [
+      ""
+    ],
+    "bin": [
+        "npiperelay.exe"
+    ],
+    "checkver": {
+        "github": "https://github.com/jstarks/npiperelay"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/jstarks/npiperelay/releases/latest/download/npiperelay_windows_amd64.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/jstarks/npiperelay/releases/latest/download/npiperelay_windows_386.zip"
+            }
+        }

```

```suggestion
        "architecture": {
            "64bit": {
                "url": "https://github.com/jstarks/npiperelay/releases/download/v$version/npiperelay_windows_amd64.zip"
            },
            "32bit": {
                "url": "https://github.com/jstarks/npiperelay/releases/download/v$version/npiperelay_windows_386.zip"
            }
        }

```
It is better to use the url that contains **version** if possible.
If we use the url for "latest". When the devs update their files, hash error will occur, until the bot catches up and update the manifest. (our bot checks for updates every 2 hours)